### PR TITLE
✏️ Fix ICE page

### PR DESCRIPTION
**Pull request by ebouchut**
- 📝 Correct the page as [**Pete suggested here 👀**](https://loop.zulipchat.com/#narrow/dm/131321,137202,489112,579198-group/near/379623172)
- 📝 Change the title of the page from `Using ICE Info` to `Meal Review`
- ✏️ Fix typos:
  - Change the name of the graph on Loop's main page from `Carbohydrates` to `Active Carbohydrates`
  - Change the name of the section from `ICE display` to `Glucose Change Display`
  - Change the Loop subpage name from `ICE` to `Carbohydrates`
- 💄 Add admonitions to make a note and a tip stand out
- 🌐  Improve automatic translation

### CODE REVIEW

**Comment by ebouchut** [comment]
@ps2 @marionbarker 
I would be happy to get your feedback on this PR.

What do you think about moving the `Insulin Counteraction Effects` section after `Glucose Change Display`?

**Review by marionbarker — COMMENTED** [review]
I think we might want to step back and review this page, carb.md page and prediction.md page as a consolidated update task. It's more work, but I think it is reasonable to make them consistent and to update the figures and models to what has been current since Loop 2 came out. Should the personal use section be removed from this page in deference to what is on the carbs.md page (Review Carb Abosrption section). Alternatively - modify the carbs.md page to point to this page. I'd like to have the how-to-use show up in only one place to make maintenance easier.

I would ask @ps to review the [Algorithm pages](https://loopkit.github.io/loopdocs/operation/algorithm/overview/) to see if there are red flags for sections on those four pages that need to be updated.

I personally do not enter multiple types of food for a single meal. I know some people do (and swear by it), but I really think a new looper might just see this as overwhelming.

File: docs/operation/features/ice.md [hunk 1]
```diff
@@ -1,30 +1,44 @@
...
 
 * food
 * stress
 * illness
 * someone sat too close to you
 
-As we all know, this list can be long but, on "normal" days FOOD will be the single strongest "insulin counteracter".  Food will be the most influence on making BGs go up.  There are also other reasons that BGs climb when we may have expected them to be steady; basals being set too low, failed infusion site, etc.
+As we all know, this list can be long but, on "normal" days food will be the strongest "insulin counteracter".  Food will be the most influence on making glucose levels go up.  There are also other reasons that glucose levels climb when we may have expected them to be steady; basals being set too low, failed infusion site, etc.
+
+If we assume that it's a "normal" day (basals are close to correct, illness is not an issue, the site is good), *Loop* will expect most of the upward pressure on glucose levels are from food.
```
Comment (ps2) [inline docs/operation/features/ice.md:17]: When you have carbs on board, Loop *always* assigns ICE to carbs, not just on a normal day. It's probably a good idea to let the user know that this is how Loop looks at it, but it's definitely not always a correct view, and that in situations where you have other positive ICE, like insulin resistance, and carbs on board, that Loop will attribute all the positive ICE to carbs, until all the entered carbs are considered absorbed.  At that point, ICE will start driving RC  upward.

Comment (ps2) [reply docs/operation/features/ice.md:17]: If you know that other non-carb effects are affecting your insulin sensitivity significantly (sickness, exercise, etc), all the info about carb absorption should be considered skewed, and not be used for trying to understand meal absorption.

**Review by ps2 — APPROVED** [review]
Added a couple of comments, but generally looks good to me. Thanks!

**Comment by marionbarker** [comment]
I decided the simplest way to offer suggestions was to make a PR to Eric's branch.
https://github.com/ebouchut/loopdocs/pull/2

**Comment by ebouchut** [comment]
Thank you @ps2 and @marionbarker for your reviews and precious help to improve this PR.

**Pull request merged**